docs/topics/mta-assessment-changes-700.adoc

```diff
@@ -3,39 +3,39 @@
 // * docs/web-console-guide/master.adoc
 :_content-type: REFERENCE
 [id="mta-assessment-changes-700_{context}"]
-= Assessment module introduction
+= The Assessment module features
 
-The *Assessment* module in {ProductFullName} offers the following functionality for assessing and analyzing applications:
+The {ProductFullName} *Assessment* module offers the following features for assessing and analyzing applications:
 
-.Assessment hub
+Assessment hub::
+The *Assessment* hub integrates with the *Application inventory*.
 
-The *Assessment* hub integrates tightly with the *Application inventory*.
+Enhanced assessment questionnaire capabilities::
+In {ProductShortName} 7.0, you can import and export assessment questionnaires. You can also design custom questionnaires with a downloadable template by using the YAML syntax, which includes the following features:
 
-.Enhanced questionnaire capabilities
+* Conditional questions: You can include or exclude questions based on the application or archetype if a certain tag is present on this application or archetype.
+* Application auto-tagging based on answers: You can define tags to be applied to applications if a certain answer was provided.
+* Automated answers from tags in application archetypes.
 
-Previously, {ProductShortName} supported a single, built-in questionnaire. {ProductShortName} 7.0 not only includes that questionnaire, but also supports importing and exporting questionnaires. You can design custom questionnaires using YAML syntax, with a downloadable template provided. The syntax provides:
++
+For more information, see xref:mta-custom-questionnaire_user-interface-guide[The custom assessment questionnaire].
 
-* Conditional questions: The syntax supports excluding or including questions based on the application or archetype if a certain tag is present on the application or archetype.
-* Application auto-tagging according to answers: The syntax supports defining tags to be applied to applications if a certain answer has been provided.
-* Automated answers from tags in application archetypes
+NOTE: You can customize and save the default questionnaire. For more information, see xref:mta-default-questionnaire_user-interface-guide[The default assessment questionnaire].
```

Comment:
Just a [quick typo fix](https://github.com/windup/windup-documentation/pull/858/commits/57b943606d4664ae1b3842b40fd668f579a12519)

Reply:
Thanks! Too many "s" for one word :-D

Reply:
It is like spelling `banana`... we all know how it starts but knowing when to finish it is tricky